snekbox/nsjail.py

```diff
@@ -163,37 +193,49 @@ def python3(
                 "--",
                 self.config.exec_bin.path,
                 *self.config.exec_bin.arg,
-                *py_args,
-                code,
-            )
+                # Filter out empty strings (causes issues with python cli)
+                *(arg for arg in py_args if arg),
+            ]
+
+            # Write files if any
+            for file in files:
+                file.save_to(fs.home)
+                log.info(f"Created file at [{(fs.home / file.name)!r}].")
```

Comment:
The brackets seem unnecessary; the path is already going to be quoted.
```suggestion
                log.info(f"Created file at {(fs.home / file.name)!r}.")
```